dashboards/google-blockchain-node-engine/metadata.yaml

```
@@ -0,0 +1,13 @@
+sample_dashboards:
+  -
+    category: Blockchain Node Engine
+    id: blockchain_node_engine
```

This ID needs to match the json/png file names.

```suggestion
    id: blockchain_node_engine_dashboard
```